README.md

```diff
@@ -1 +1,22 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse!.
+
+
+## Built With
+
+- HTML/CSS
+- Git/Github/Linter
+
+## Authors
+
+👤 **Paul Sáenz Sucre**
+
+- GitHub: [@elalfildecolor](https://github.com/elafildecolor)
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
- [ ] Please add a descriptive README file to your project by using [this template](https://github.com/microverseinc/readme-template) and customize it. Please make sure to add and keep only the main header section and do not keep them empty. Also add a license in order to make it more professional :+1: 
- [ ] Your project doesn't have a license. I would like to recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish and add the licence link to section in your readme dedicated for it in order to keep it more professional :+1: